src/cmd/main.cpp

```diff
@@ -70,7 +71,8 @@ int main(int argc, const char* argv[]) {
         "Specifies port to bind to, if not specified in address")(
         "loadconfig,l", "Loads configuration if set (deprecated)")(
         "synctype,s", po::value<unsigned int>(&synctype), synctype_descr)(
-        "recovery,r", "Runs in recovery mode if set");
+        "recovery,r", "Runs in recovery mode if set")(
+        "logpath,g", po::value<string>(&logpath), "customized log path");
```

Comment:
Can we make it clear in the help message what kinds of paths are supported (relative or absolute or both)